### PR TITLE
Homework_12

### PR DESCRIPTION
**Pull request by vmelfx**
Homework 12 from git.

### CODE REVIEW

File: Lecture_12/meleshchenko_task_n1_hw_12.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        num += 1
+
+
+# Be careful to run this!!!
+# for element in infinite_sequence():
+#     print(element)
+
+# Example 3. Numeric palindrome detector
+
+def palindrome_detector(num):
```
Comment (Ruslan-Skira) [inline Lecture_12/meleshchenko_task_n1_hw_12.py:40]: type hinting

File: Lecture_12/meleshchenko_task_n1_hw_12.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    """
+    This function takes an input number, reverses it,
```
Comment (Ruslan-Skira) [inline Lecture_12/meleshchenko_task_n1_hw_12.py:42]: Please write in the docstring 
1. what attributes do you use and what type of them?
2. what function returns
3. How to use this function example.

**Review by Ruslan-Skira — COMMENTED** [review]
Nothing to add in major. Good work,  continue.

**Comment by vmelfx** [comment]
Docstring changed to.

**Review by Ruslan-Skira — APPROVED** [review]

**Pull request merged**